lib/Twig/Extension/Core.php

```diff
@@ -1116,7 +1116,7 @@ function _twig_escape_css_callback($matches)
     // \xHH
     if (!isset($char[1])) {
         $hex = ltrim(strtoupper(bin2hex($char)), '0');
-        if (0 === strlen($hex)) {
+        if ('' === $hex) {
```

Comment:
>Have you some numbers to back this change?

no, the rationale behind is that it would save a function call (but I didn't check the implementation of `strlen` in a long time)

> In any case, I think the different is really small and that it's just not worth it.

agreed, reverted :)